nuxeo-platform-spreadsheet-web-ui/pom.xml

```diff
@@ -0,0 +1,15 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.nuxeo.ecm.platform</groupId>
+    <artifactId>nuxeo-platform-spreadsheet</artifactId>
+    <version>8.10-SNAPSHOT</version>
+    <relativePath>../pom.xml</relativePath>
+  </parent>
+
+  <artifactId>nuxeo-platform-spreadsheet-web-ui</artifactId>
+  <name>Nuxeo Platform Spreadsheet Web UI</name>
+  <description>Nuxeo Platform Spreadsheet Web UI</description>
```

Comment:
No dependencies? Doesn't it depend on `nuxeo-platform-spreadsheet-core` and  `nuxeo-platform-spreadsheet-web` at least?